### PR TITLE
AB#792 Plan and implement Redux architecture

### PR DESCRIPTION
**Pull request by pmakowski23**

### CODE REVIEW

File: src/redux/api/messages.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+   * Page number
+   *
+   * @default 1
+   */
+  page?: number;
+  /**
+   * Category number used to filter results.
+   *
+   * @default
+   * 0 - "Ogólne"
```
Comment (JKrymarys) [inline src/redux/api/messages.ts:15]: I am really bothered with `* 0 - "Ogólne"`, but I guess that it's some example of api response, right?

File: src/common/types/messages.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+export type Messages = {
```
Comment (JKrymarys) [inline src/common/types/messages.ts:1]: It's not only messages. Let's name is sth like "MessageResponse" or sth that will make it clear that this object has messages + sth else (info about pagination)

**Review by JKrymarys — APPROVED** [review]

File: src/redux/api/messages.ts (hunk 1 above)
Comment (pmakowski23) [reply src/redux/api/messages.ts:15]: Default is 0 and it stands for category named "Ogólne" at least I think I interpreted that right from the wiki. I don't know how to describe that differently so if you have an idea I'm all ears 😁

File: src/common/types/messages.ts (hunk 2 above)
Comment (pmakowski23) [reply src/common/types/messages.ts:1]: So maybe with this logic I should name `Accounts` and  `Pharmacies` AccountResponse and PharmacyResponse?

**Pull request merged**